SPECS/openblas/openblas.spec

```diff
@@ -254,7 +230,7 @@ rm -rf lapack-netlib
 cd ..
 cp -ar OpenBLAS-%{version} openmp
 cp -ar OpenBLAS-%{version} threaded
-%if %build64
+%if %{build64}
```

Comment:
To be honest, we can also assume that we're always building for a 64-bit architecture and remove the `build64` macro completely. Not a blocker, completely ignorable for this PR, though.